site/docs/spark-queries.md

```diff
@@ -234,6 +234,189 @@ SELECT * FROM prod.db.table.manifests
 +----------------------------------------------------------------------+--------+-------------------+---------------------+------------------------+---------------------------+--------------------------+---------------------------------+
 ```
 
+### Metadata Table Schema
```

Comment:
Not sure what other people think, I think I would prefer having the schema all in that `metadata.md` page, and we can remove this section in Spark and only provide a SQL example for the syntax to query the table + a link to that page.

Reply:
I don't see much value in having these schemas here. Seems like they'll just get out of date.